### PR TITLE
add 6.4.0 changelog

### PR DESCRIPTION
**Pull request by hzoo**
Will need to add to it if we decide to merge anymore PRs

### CODE REVIEW

**Comment by amasad** [comment]
Looks good!

**Comment by hzoo** [comment]
updated with the latest 2 PRs

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,45 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
...
+ * `babel-generator`: [#3229](https://github.com/babel/babel/pull/3229) Check for parentheses for `AwaitExpressions` and fix over-parentheses in `YieldExpressions`.
+
+* **Spec Compliancy**
+ * `babylon`: [#3225](https://github.com/babel/babel/pull/3225) throw parse error if class properties do not have a semicolon.
+
+* **Breaking Change**
+ * `babel-plugin-external-helpers`: [#3205](https://github.com/babel/babel/pull/3205) Renamed from `babel-plugin-external-helpers-2` due to someone taking the npm name beforehand.
+
+* **Documentation**
+ * `babel-generator`: [#3240](https://github.com/babel/babel/pull/3240) Fix small in babel-generator README sample code. Thanks @athaeryn!
```
Comment (jmm) [inline CHANGELOG.md]: @hzoo Typo about a typo :)

Comment (hzoo) [inline CHANGELOG.md]: sad day :frowning:

**Pull request merged**